source/misc/bash_startup.in

```diff
@@ -1,257 +1,95 @@
-#!/bin/bash
-# This is based on "preexec.bash" but is customized for iTerm2.
+# This is inspired by "preexec.bash" but is customized for iTerm2. Sourced, so
+# no shebang. This module requires 2 bash features you must not otherwise be
+# using: the DEBUG trap and the PROMPT_COMMAND variable.
 
-# Note: this module requires 2 bash features which you must not otherwise be
-# using: the "DEBUG" trap, and the "PROMPT_COMMAND" variable.  iterm2_preexec_install
-# will override these and if you override one or the other this _will_ break.
+osc='\033]'; cso='\007' # Beginning, ending escape sequences.
```

Comment:
No, it's actually an OSC (Operating System Command). `ESC + ]` = OSC, per ECMA 48 and various other standards. I use the xterm docs as the most convenient almanac for this stuff: http://invisible-island.net/xterm/ctlseqs/ctlseqs.html